spec/models/organization_spec.rb

```diff
@@ -6,6 +6,26 @@
   it { is_expected.to have_many(:sponsorships) }
   it { is_expected.to have_many(:organization_memberships).dependent(:delete_all) }
 
+  describe "#after_commit" do
+    it "on update syncs elasticsearch data" do
+      article = create(:article, organization: organization)
+      sidekiq_perform_enqueued_jobs
+      new_org_name = "#{organization.name}+NEW"
+      organization.update(name: new_org_name)
+      sidekiq_perform_enqueued_jobs
+      expect(article.elasticsearch_doc.dig("_source", "organization", "name")).to eq(new_org_name)
+    end
+
+    it "on destroy removes data from elasticsearch" do
```

Comment:
`it "removes data from Elasticsearch on destroy"`

Reply:
lol I think these will be consolidated during the refactoring, I will update the wording then 😄